caf-logging-serilog/caf-logging-serilog.sln

```diff
@@ -11,8 +11,8 @@ Global
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{540E19A9-A196-4BFC-8F92-7F55608726F0}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
https://docs.microsoft.com/en-us/visualstudio/debugger/how-to-set-debug-and-release-configurations?view=vs-2022